dewy/chunks/router.py

```diff
@@ -64,5 +71,6 @@ async def retrieve_chunks(
     return RetrieveResponse(
         summary=None,
         text_results=text_results if request.include_text_chunks else [],
+        image_results=[],
```

Comment:
Required field

Reply:
I believe we could set a default in the response if we wanted. But this is more explicit, and will likely better align to the implementation we eventually have anyway.